### DISCUSSION TITLE
Threads.net authentication

### DISCUSSION THREAD

**Discussion by atumas-bananamilk** [nextauthjs/next-auth#11230] [Ideas]
### Goals

1. Signup / Login using Threads.net

### Background

Threads.net have just released their API.
It'd work similar to Facebook, Twitter login, but for Threads.
This is really needed by our customers.

### Proposal

N/A

**Comment by k-taro56** [comment]
We just released.
https://github.com/nextauthjs/next-auth/releases/tag/%40auth%2Fcore%400.34.0

Thank you.
